content/en/apps/features/integrations/android.md

```diff
@@ -17,6 +17,84 @@ When the user clicks on a link to a CHT instance from an SMS, email, WhatsApp, o
 
 Users can choose "Always" to skip this prompt in future. The prompt may look different depending on the version of Android being used.
 
+## Android App Links verification
+
+Starting with Android 12, Android supports associating an app with a domain and automatically verify this association
+to allow deep links to immediately open the app without requiring your user to select the app.
+To get this working, you need to host a Digital Asset Links JSON file at `https://<domain.name>/.well-known/assetlinks.json`
+containing some information about your app to associate it with your domain.  
+More information available on the [official Android docs](https://developer.android.com/training/app-links/verify-android-applinks).
```

Comment:
The line breaks were to make the section fit my screen without having to scroll horizontally :sweat_smile:  I figured since it's markdown it doesn't really matter if we have line breaks mid-sentence, it'll still render as a regular space between words